doc/kci_data.md

```diff
@@ -125,34 +125,32 @@ having an API token for the KernelCI staging API.  This was useful to be able
 to show the results on the web dashboard.  The same command can be run with a
 local instance using `--db-config=localhost`.
 
-## Creating user
+## Creating new API user (experimental)
 
-We can create new users of KernelCI-API using `kci_data`.
+We can create users for the new [KernelCI API](/docs/api) using `kci_data`.
+This is still under development and it's not used in production by KernelCI.
 
-We need to provide an API token to access `/user` endpoint of API.
-Please refer the instructions [here](https://kernelci.org/docs/api/api-details/#create-an-api-token-with-security-scopes) to create token for admin user.
+An API token with admin permissions is required in order to do this, please see
+the documentation about [how to create
+one](/docs/api/api-details/#create-an-api-token-with-security-scopes).
 
-For security purpose, instead of providing `--db-token` from the comman line we can add `db_token` in settings file i.e. `kernelci.conf`.
+For security reasons, instead of providing `--db-token` from the command line
+we can add `db_token` in the [settings file](../settings) e.g. `kernelci.conf`:
 
-```
+```ini
 [db:api]
 
 db_token: XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
 ```
 
-> **Note** Only admin users are permitted to use this utility of creating users.
-
-
-Command to create a regular user would be:
+Here's a sample command to create a regular user:
 
 ```
 ./kci_data create_user --username test --password test --db-config api
-{'_id': '628c81e641616ced4e31a10b', 'username': 'test', 'hashed_password': '$2b$12$Ev72ythf0jViigUOtHuifOW6Mtrr0RmwQx0zO1CIy5XMwrN57bhuW', 'active': True, 'is_admin': False}
```

Comment:
Shouldn't we include what the response would look like?

Reply:
No because I don't think `kci_data` should be printing this, it looks more like debugging data to me. `curl` prints it because it's a low-level tool for doing HTTP queries, but `kci_data` is a command line tool for KernelCI users.

Reply:
Typically, if it worked then nothing needs to be printed. That's how Unix command line tools work. Maybe we can have a verbose mode to print something like "User created.". Then if there's an error then yes it should be printed of course.